workos/user_management.py

```diff
@@ -160,27 +133,44 @@ def list_users(
 
         return self.construct_from_response(response)
 
-    def delete_user(self, user_id):
-        """Delete an existing user.
+    def create_user(self, user):
+        """Create a new user.
 
         Args:
-            user_id (str) -  User unique identifier
+            user (dict) - An user object
+                user[email] (str) - The email address of the user.
+                user[password] (str) - The password to set for the user. (Optional)
+                user[first_name] (str) - The user's first name. (Optional)
+                user[last_name] (str) - The user's last name. (Optional)
+                user[email_verified] (bool) - Whether the user's email address was previously verified. (Optional)
+
+        Returns:
+            dict: Created User response from WorkOS.
         """
-        self.request_helper.request(
-            USER_DETAIL_PATH.format(user_id),
-            method=REQUEST_METHOD_DELETE,
+        headers = {}
+
+        response = self.request_helper.request(
+            USER_PATH,
+            method=REQUEST_METHOD_POST,
+            params=user,
+            headers=headers,
```

Comment:
ooc what was this change (adding empty headers) for? or was it copy/paste?

Reply:
This was me moving methods around (so they are in a different order), and the diff just not dealing with that well 🙃

Reply:
oooooh yeah i see my bad 😆